src/components/AppletsTable/Table/AppletItem/AppletItem.styles.ts

```diff
@@ -0,0 +1,6 @@
+import { Box, styled } from '@mui/material';
+
+export const StyledAppletName = styled(Box)`
```

Comment:
here you can use StyledFlexTopCenter global component from styledComponent folder